src/common/states.ts

```diff
@@ -165,4 +165,35 @@ export const STATES_DATA = {
 
 export type State = keyof typeof STATES_DATA;
 
+export type StateCode = State;
+
+export type StateName =
```

Comment:
You could write something like this to get the state name from `STATES_DATA` without needed to rewrite everything again.

- add `as const` after `STATES_DATA` declaration
- extract values from `STATES_DATA` object with `type StateName = typeof STATES_DATA[keyof typeof STATES_DATA]['name'];`

reference: https://www.typescriptlang.org/play/?ssl=166&ssc=71&pln=166&pc=1#code/MYewdgzgLgBAygFQIIIKJwPoBEVJgXhgG8AoGGJAYQC5izyZQATAU1oHIAmdgGnvICGAJxYDKIVhFoBtAGwAOALp8GMAJYsAMizABzKAAtaARgDMKhmAEBbNjHZJgI3vQC+FpJtqlVzO+wAWF1VhUXFJGXlOZX51LR19IxgATgtyK1sOJAAbAV0QAQhgmHd6JAAFb1i-Dm40mFCxCRYpGGlk2RjVDW09Q1pU2Iz-JGsBAAcAQ+LS8iQAWSrfZtriwREmiLbkzh4UgHYuhh6E-pT64ayxgC9wQpmLACEkJYYa+wBWNYaN8JaZfbGPb7cwwfYBYEfYGHeonPpJaTyPbJI7pGz+R4CAxqAQPeiUVCvcjvdimb6NP6tRFQmDyJSw+LwgYXdEcSiiITTCyzGBYABiRMYK3sxnJv2aVNkxlRcV6iRM5iGrPsWDU0CEaigIBgfJYrCEAmyePI6EFJP2YrCEpknH2e049NicPl5yVmXsqAg4wAtxqtfABGAtcaYABxADyZuF7FF9Qp1rasl2MFkARlzrOg1Ul3soZAakmRW5FnmLzoy1YHDJcfFW3aSJSKIZcszLPd7HmwkDBgAxyAQ-NQ1HK-ZZJbNv82qYgTBTMnTKDTBDZzTTHbZ47uoyXWY2-55morBAwywDWqBwhh-5Y7F43XZDTZJ1m6cksYgW79wJ-aGhCAIBA-bFvQ8xwFeHA3iEtaTnIMJOturafhwnY-n+AHakw2pwAArkawHkOUZY+G80Z1Le0FUskM7JKCyRpi+TKutmyrsOUwhcm4FjlI84H2EENZWnW8imOmCFJFmlgsWxBregARri+EwOUhLliRI6BOOlKRDO8hwVuLbiXuHDlKeGSydhoAhuUACSvGkppCaIrIezyE28EGcySH2OUOLYd6VkAEp2ckDl1gEM4BMmASggEy4BDSATPu5r4mAADEZ3ldmAHHkDyAUAFJ2RaAkTlSnAzpwyacPRyWMUiXnsAFagYSwMD5YGLBqH+IYBQAcnZ-HkYJMHyDV+kpdsezGKlMo5o1zVhgaYCsDAmEwL1IBCFALA9ZGqnEqRoUwbIbnjYxxgQg1AXgEwAAvYA4j1YH7UK6mpUdVIfDOHygh8y4fB8okeTA00ZfN2q-oGK1rTheGcfQAVBS9JJkVBw2UYDDEuhJaLttdZ5jCGcA0Mj0YhSVWltAE64BA2ASnccYmecx7ZwIGUACDAlDfsIh4KfD5BwCpxEHepXwU45+wM+QGaGQ1cCnroajjDtilwJUpPqfIH0yO+U3JruIPLsYNLGM5IPrsYDbGNLsoTcYuzy32SkCLhQECzACB7SLr3XjriYiVjZztFN0pgwgIDAOzh5FvDrgNMeoCQFAADcJAkFAACeKvwBz229eiBAwFnKsgAAZvAyBoJgODINIADWLCZ+XxfZywLeICg6DYLgijSOwwzsIoKdAA

Reply:
I love it! I didn't know how to do that

Reply:
It's not possible because break "inscrição estadual"

![image](https://user-images.githubusercontent.com/16626980/83975897-bdd6ff00-a8cc-11ea-928f-9381fd11e7cc.png)

Reply:
hmmm... maybe its "insrição estadual" issue :(